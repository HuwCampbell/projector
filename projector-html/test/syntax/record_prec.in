{ foo.bar baz.quux wem.quib pil mun }