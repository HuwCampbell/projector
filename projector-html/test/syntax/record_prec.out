Template
  ()
  Nothing
  (TENode
     ()
     (THtml
        ()
        [ TExprNode
            ()
            (TEApp
               ()
               (TEApp
                  ()
                  (TEApp
                     ()
                     (TEApp
                        ()
                        (TEPrj () (TEVar () TId { unTId = "foo" }) TId { unTId = "bar" })
                        (TEPrj () (TEVar () TId { unTId = "baz" }) TId { unTId = "quux" }))
                     (TEPrj () (TEVar () TId { unTId = "wem" }) TId { unTId = "quib" }))
                  (TEVar () TId { unTId = "pil" }))
               (TEVar () TId { unTId = "mun" }))
        ]))